tests/Fixtures/Integration/misc/PHP8_3.test

```diff
@@ -0,0 +1,116 @@
+--TEST--
+PHP 8.3 test.
+
+@TODO Fix whitespace inside dynamic const fetch, e.g. `Foo::{  $bar  }`
+@TODO Fix whitespace between const type and const name, e.g. `public const string       TEST = 'Test1';`
```

Comment:
@TODO PHP 8.3 Fix whitespace between const type and const name, e.g. `public const string       TEST = 'Test1';`

Reply:
externalised https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7446